CMakeLists.txt

```diff
@@ -7,6 +7,13 @@ catkin_package()
 set(ROS_VERSION "1")
 set(ROS_DISTRO "melodic")
 
+set(ROS_PY_VER "$ENV{ROS_PYTHON_VERSION}")
+if(ROS_PY_VER)
+  set(ROS_PYTHON_VERSION "${ROS_PY_VER}")
```

Comment:
I would suggest to change this to the following to avoid the temporary variable:

```
if(DEFINED ENV{VAR})
  set(ROS_PYTHON_VERSION "$ENV{ROS_PYTHON_VERSION}")
```